### PR TITLE
#755 Pošiljanje SMS: indikator stanja in cene

### PR DESCRIPTION
**Pull request by KlemenSpruk**
- user can view notifications license state
-  license report was added
- user is notified when he has not credit left

### CODE REVIEW

File: django_project_base/notifications/models.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ class AbstractDjangoProjectBaseMessage(models.Model):
 
     class Meta:
         abstract = True
-        verbose_name = "Notification"
+        verbose_name = "Notification Message"
```
Comment (velis74) [inline django_project_base/notifications/models.py:32]: notification ALI message, ne pa oboje.

File: django_project_base/notifications/base/channels/mail_channel.py [hunk 2]
```diff
@@ -10,7 +10,7 @@
 class MailChannel(Channel):
     id = ChannelIdentifier.MAIL.value
 
-    name = "E-Mail"
+    name = "EMail"
```
Comment (velis74) [inline django_project_base/notifications/base/channels/mail_channel.py:13]: to ni za prevajat?

File: django_project_base/licensing/logic.py [hunk 3]
```diff
@@ -37,9 +38,15 @@ def report(self, user: Model) -> dict:
         usage_report = []
         for agg in used:
             if content_type := ContentType.objects.get(pk=agg["content_type"]):
-                usage_report.append({"item": content_type.model_class()._meta.verbose_name, "price": agg["count"]})
+                usage_report.append({"item": content_type.model_class()._meta.verbose_name, "usage_sum": agg["count"]})
+        used_credit = used.first()["count"]
         return LicenseReportSerializer(
-            {"available_credit": MONTHLY_ACCESS_LIMIT_IN_CURRENCY_UNITS, "used_credit": 3, "usage_report": usage_report}
```
Comment (velis74) [inline django_project_base/licensing/logic.py:42]: Monthly? A ma Janez zdej vsak mesec 100€ za pokurt?

**Review by velis74 — CHANGES_REQUESTED** [review]

File: django_project_base/notifications/models.py (hunk 1 above)
Comment (KlemenSpruk) [reply django_project_base/notifications/models.py:32]: popravljeno

File: django_project_base/notifications/base/channels/mail_channel.py (hunk 2 above)
Comment (KlemenSpruk) [reply django_project_base/notifications/base/channels/mail_channel.py:13]: ne

File: django_project_base/licensing/logic.py (hunk 3 above)
Comment (KlemenSpruk) [reply django_project_base/licensing/logic.py:42]: za sedaj ni casovne omejitve

**Pull request merged**